doc/autoyast.md

```diff
@@ -1,141 +1,101 @@
-  partition table type to use, as the current proposal does) can also be a
-  reasonable option.
-
-### Handling LVM
-
-The approach to LVM of the level 3 of AutoYaST and the current proposal is
-totally different.
-
-In a normal installation, there is only one VG (that can be a reused one) and
-the number and sizes of the PVs are calculated dynamically based on the needs
-defined by the `PlannedLv` objects and the restrictions imposed by the
-`PlannedPartition` ones. In other words, not all the planned partitions are
-known at the beginning of the process, those partitions that will become PVs are
-planned while generating the resulting devicegraph. All the LVM-related logic is
-encapsulated in a class called `LvmHelper`.
+## The AutoYaST level 3 procedure
 
-With AutoYaST the process is more straightforward. The exact list of PVs is
-specified in the AutoYaST profile. On the other hand, there can be several VGs
-involved.
+### Phase one: converting the profile data into proper objects
 
-It makes sense to break the current `LvmHelper` into two components. The first
-one will generate `PlannedPartition` objects as described (to become PVs) and
-also an object of a new `PlannedVg` class. The second component, that should
-probably be called `LvmCreator` for consistency reasons, could take collections
-of planned partitions (PVs), planned VGs and planned LVs as input and create
-the corresponding structures in the target devicegraph. As a bonus, that would
-be more consistent with the rest of the current proposal mechanisms.
+As you may know, an AutoYaST profile is basically an XML file which is converted
+into a complex hash, including nested hashes and arrays. Working with such a
+data structure is not convenient at all so, as a first step, the partitioning
+section of the profile is turned into an
+{Y2Storage::AutoinstProfile::PartitioningSection} object, which offers a a
+better API for this use case.
 
-The new `PlannedVg` and `LvmCreator` classes will be re-used by the AutoYaST
-proposal and will have the capabilities needed to honor the following properties
-from the AutoYaST profile: `pesize`, `lv_name`, `stripes`, `stripe-size`,
-`lvm_group`, `pool`, `used_pool` and `keep_unknown_lv`.
+This class does not work in isolation, as each element of the profile is
+represented by a different class ({Y2Storage::AutoinstProfile::DriveSection},
+{Y2Storage::AutoinstProfile::PartitionSection}, etc.).
 
+As a side note, the {Y2Storage::AutoinstProfile::PartitioningSection} class is
+used when cloning a system too. Check the
+{Y2Storage::AutoinstProfile::PartitioningSection.new_from_storage} method for
+further details.
 
-## The AutoYaST level 3 procedure
-
-### Phase one: assigning the drives
+### Phase two: assigning the drives
 
 The `partitioning` section of the AutoYaST profile is organized into drives
 containing a list of partitions each. The result must honor that organization.
```

Comment:
I know the text was already there, but the following "I.e., two partitions..." was likely meant to be "E.g., two partitions..."